modprobe.conf/modprobe.conf.common

```diff
@@ -71,18 +71,10 @@ blacklist pata_acpi
 softdep uhci-hcd pre: ehci-hcd
 softdep ohci-hcd pre: ehci-hcd
 
-# libcrc32c calls crypto_alloc_shash("crc32c", 0, 0), which results in a
-# request_module("crc32c"), but that dependency is not seen by modpost/depmod
-# https://bugzilla.novell.com/552443
-# SUSE INITRD: libcrc32c REQUIRES crc32c
-
-# likewise, cifs calls crypto_alloc_shash(...) for a couple of algorithms
-# https://bugzilla.novell.com/730617
-# SUSE INITRD: cifs REQUIRES des ecb md4 md5 hmac arc4
-# It also calls load_nls()
+# cifs calls load_nls()
 # FIXME: determine the proper nls module based on fstab options instead
-# SUSE INITRD: cifs REQUIRES nls_utf8
- 
+softdep nls pre: cifs: nls_utf8
```

Comment:
This looks wrong. Did you mean `softdep cifs pre: nls_utf8` ?

Reply:
yes, that sounds like it. 
at any rate cifs: is invalid syntax and nls is built-in so the rule would never apply

Reply:
I've reverted this hunk after the merge.